src/google-analytics-tracker.ts

```diff
@@ -0,0 +1,37 @@
+import IAnalyticsTracker from './analytics-tracker';
+import Metric from './metric';
+
+declare global {
+  // tslint:disable-next-line:interface-name
+  interface Window {
+    ga: any;
+  }
+}
+
+export default class GoogleAnalyticsTracker implements IAnalyticsTracker {
+  name: string = 'Google Analytics';
+  metricQueue: Metric[] = [];
+  timingVar: string = 'name';
+
+  canSend(): boolean {
```

Comment:
For `boolean` type we already use the prefix convention `is`, can you name this method `isReady`.
Because is a `read-only` variable, we can use a Typescript Getter instead.
```
get canSend(): boolean {
   return window.ga && typeof window.ga === 'function';
}
```

Reply:
This will be removed anyway in upcoming simplification refactor.